cicecore/drivers/nuopc/cmeps/CICE_RunMod.F90

```diff
@@ -86,8 +86,6 @@ subroutine CICE_Run
       call init_flux_atm  ! Initialize atmosphere fluxes sent to coupler
       call init_flux_ocn  ! initialize ocean fluxes sent to coupler
 
-      call calendar()       ! at the end of the timestep
```

Comment:
Is this line causing problem?

Reply:
Yes. The call calendar was added to the end of the new advance_timestep routine so no longer needs to be called here.